docs/Project.toml

```diff
@@ -1,6 +1,7 @@
 [deps]
 BenchmarkTools = "6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf"
 CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
+DataFlowTasks = "d1549cb6-e9f4-42f8-98cc-ffc8d067ff5b"
```

Comment:
I never add the project itself as a dependency to `Documenter`, as I was under the impression this was not necessary (`Documenter` looks into its parent folder for a project). Has this changed?

Reply:
I don't think `Documenter` looks into its parent folder to find the documented project, so at the time when the documentation is built, the parent package has to be present in the documentation environment in some way. However, one of the steps in the GitHub CI action takes care of `Pkg.dev`ing the parent project in the documentation subproject. So both ways work: if the committed `docs/Project.toml` does not mention `DataFlowTasks`, things will get set up correctly just ahead of time.

I've recently seen packages where even the `docs/Manifest.toml` was committed so as to make sure that the parent project is `dev`ed from the parent directory. [`Graphs.jl`](https://github.com/JuliaGraphs/Graphs.jl/tree/master/docs) is an example of this.

Personally, I'd be in favor of keeping `DataFlowTasks` as a documentation dependency in the committed `docs/Project.toml`, because this simplifies the process to locally build the documentation. But I'm open to compromise :smile: